src/styles.css

```diff
@@ -36,17 +38,17 @@ input {
   border-radius: 5px;
   padding: 0 20px;
 
-  -webkit-box-shadow: 2px 2px 3px 2px #111;
-  -moz-box-shadow: 2px 2px 3px 2px #111;
-  box-shadow: 2px 2px 3px 2px #111;
+  -webkit-box-shadow: 2px 2px 4px 0px #111;
+  -moz-box-shadow: 2px 2px 4px 0px #111;
+  box-shadow: 2px 2px 4px 0px #111;
```

Comment:
Tu nie wydupia zaś błędów, że z wielkich liter?

Reply:
to nie o to chodzilo, tylko o style w plikach jsowych, tam jest konwencja camel case, a te style wyjatkowo maja byc pascal casem (ale tylko w jsx, nie w cssach)